### PR TITLE
created email verify page with logic and updates on forgot password

### PR DESCRIPTION
**Pull request by deviator53**
- updated forgot password page
- created verify page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@mui/styles": "^5.9.2",
     "@reduxjs/toolkit": "^1.8.4",
     "@types/react-redux": "^7.1.24",
+    "axios": "^0.27.2",
```
Comment (taiwoAkindele) [inline package.json:22]: Why are you using axios? Did you check with Brain before installing it?

File: src/env.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
```
Comment (taiwoAkindele) [inline src/env.d.ts]: This is not meant to be here. please move it to the root of your folder

**Comment by taiwoAkindele** [comment]
Please resolve the conflicts on your branch

**Review by Vinyl-Davyl — COMMENTED** [review]
There are some conflicts on this branch, pls resolve

File: src/env.d.ts (hunk 2 above)
Comment (Vinyl-Davyl) [reply src/env.d.ts]: Not sure we are meant to push the .env file either. Should be in .gitignore

Comment (pablo-clueless) [reply src/env.d.ts]: If you move it to the root, it will be ignored, it was included in the .gitignore.

**Review by taiwoAkindele — COMMENTED** [review]
you have still not removed the env file from your pr

**Comment by deviator53** [comment]
@taiwoAkindele should i delete it, because when i put it in the root folder it flags error

**Comment by taiwoAkindele** [comment]
> @taiwoAkindele should i delete it, because when i put it in the root folder it flags error

Not don't delete just move it there. It will still work

**Comment by deviator53** [comment]
@taiwoAkindele I have moved it

**Review by taiwoAkindele — APPROVED** [review]

**Review by PhilemonBrain — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (PhilemonBrain) [reply package.json:22]: @deviator53 u didnt answer this question... i thought we have a hook for making requests already

**Pull request merged**